bin/release-check/find-latest-releases.sh

```diff
@@ -28,11 +28,11 @@ case $product_type in
 esac
 
 # SEARCH PAC
-pac_search_query="https://packages.atlassian.com/api/search/versions?g=${GROUP}&a=${ARTIFACT}&c=&repos=maven-closedsource-local"
+pac_search_query="https://packages.atlassian.com/maven/${GROUP}/${ARTIFACT}/maven-metadata.xml" #get xml
 search_response=$(curl -s "${pac_search_query}")
 
 # PARSE RESPONSE AND GET AN ARRAY OF VERSIONS THAT CONTAIN ONLY NUMBERS (EXCLUDES SNAPSHOTS AND MILESTONE RELEASES)
-latest_releases=$(echo "$search_response" | jq -r '[ .results[].version | select(. | test("^\\d+\\.\\d+.\\d+$")) ]')
+latest_releases=$(echo "$search_response" | grep -oE '<version>[0-9]\.[0-9]+\.[0-9]+</version>' | grep -oE '[0-9]\.[0-9]+\.[0-9]+')
```

Comment:
I think the first digit of the version could be `[0.9][0.9?]....` so it would exclude the `1000.x` version and yet be future proof for Jira 10, 11.

Reply:
You are right. I changed regex to get versions 10 and higher.